src/NzbDrone.Core/Extras/ExtraService.cs

```diff
@@ -48,9 +52,32 @@ public class ExtraService : IExtraService,
             _diskProvider = diskProvider;
             _configService = configService;
             _extraFileManagers = extraFileManagers.OrderBy(e => e.Order).ToList();
+            _albumExtraManager = albumExtraManager;
             _logger = logger;
         }
 
+        /// <summary>
+        /// Import extra files belonging to an album. Track-specific extras will be ignored.
+        /// </summary>
+        /// <param name="importedTracks">The imported tracks in the source dir.</param>
+        /// <param name="destinationDir">Destination dir for the album extras</param>
+        public void ImportAlbumExtras(List<ImportDecision<LocalTrack>> importedTracks, string destinationDir)
+        {
+            var firstTrack = importedTracks.First();
+            var artist = firstTrack.Item.Artist;
+            var albumId = firstTrack.Item.Album.Id;
+
+            var sourceFolder = _diskProvider.GetParentFolder(firstTrack.Item.Path);
+            var albumFiles = _diskProvider.GetFiles(sourceFolder, false);
+            var filtered = albumFiles.Where(x => !importedTracks.Select(t => t.Item.Path).Contains(x));
+
+            // TODO: grab extensions from config?
+            var extensions = new List<string> { ".cue", ".jpg", ".png", ".nfo", ".log", ".txt" };
```

Comment:
_configService.ExtraFileExtensions

or a new setting if we think it's needed (existing for Artist and new for Album extensions)

Reply:
I have two questions regarding this suggestion:
1. The default value currently is `srt`, which is probably coming from the TV Arrs. 
-> Shall we define a list of suitable extensions, e.g. the ones I did my tests with and hardcoded above?
2. The setting is currently off by default.
-> I was hoping that this feature can be instead on / opt out.

Especially wrt the second point - the impact of downloading dozens of albums and finding out afterwards that none of the usual extra files (nfos, covers, cue's) are there is huge. I literally had to stop my noise file acquisition for weeks until I figured out what's happening and came up with this PR. I'm still slowly going through my spreadsheet and rebuilding artist by artist.

With that said, I assumed that this should be a different setting, which does not seem to make sense. Putting this under the umbrella of "Extra Files" is fine.

Reply:
1. Yes, let's change the default to something that makes sense. SRT doesn't (I'm actually surprised it's gone this long)
2. I'm not opposed to on by default if we think that makes sense for music.

Reply:
Alright, I made the changes. The defaults are as following: 
- Import extra files on by default;
- Extra file extensions: `log, cue, nfo, jpg, jpeg, png`

Regarding sensible defaults for the extensions:
- Must-haves are `.log` and `.cue`: highly valued in at least one particular music _community_ :)
- Nice to haves: `.nfo` (release info)
- Maybe: `.pdf` and `.txt` (release-specific, some include such extra files or docs)
- Whatever: cover files (so your jpg's, png's etc), this is recoverable information

I could add all of these, but I didn't want to make the list too long. Let me know what you think.